Tutorials/Cache-Static-Assets/index.md

```diff
@@ -0,0 +1,99 @@
+using System.IO;
+using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Core.Hosting;
+using Umbraco.Extensions;
+
+public class StaticFilesComposer : IComposer
+{
+    public void Compose(IUmbracoBuilder builder)
+        => builder.Services.AddTransient<IConfigureOptions<StaticFileOptions>, ConfigureStaticFileOptions>();
+
+    private class ConfigureStaticFileOptions : IConfigureOptions<StaticFileOptions>
+    {
+        private readonly IHostingEnvironment _hostingEnvironment;
+        private readonly GlobalSettings _globalSettings;
+        private static readonly HashSet<string> _fileExtensions = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        {
+            ".ico",
+            ".css",
+            ".js",
+            ".svg",
+            ".woff2"
+        };
+
+        public ConfigureStaticFileOptions(IOptions<GlobalSettings> globalSettings, IHostingEnvironment hostingEnvironment)
+        {
+            _globalSettings = globalSettings.Value;
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        public void Configure(StaticFileOptions options)
+            => options.OnPrepareResponse = ctx =>
+            {
+                string backOfficePath = _globalSettings.GetBackOfficePath(_hostingEnvironment);
+
+                // Exclude Umbraco backoffice assets
+                if (ctx.Context.Request.Path.StartsWithSegments(backOfficePath))
+                {
+                    return;
+                }
+
+                // Set headers for specific file extensions
+                var fileExtension = Path.GetExtension(ctx.File.Name);
+                if (_fileExtensions.Contains(fileExtension))
+                {
+                    var headers = ctx.Context.Response.GetTypedHeaders();
+
+                    // Update or set Cache-Control header
+                    var cacheControl = headers.CacheControl ?? new CacheControlHeaderValue();
+                    cacheControl.Public = true;
+                    cacheControl.MaxAge = TimeSpan.FromDays(365);
+                    headers.CacheControl = cacheControl;
+                }
+            };
+    }
+}
+```
+
+## Cache images in ImageSharp
+
+For setting `Cache-Control` max-age header for images processed by the ImageSharp middleware, you can set the `Umbraco:CMS:Imaging:Cache:BrowserMaxAge` setting.
+
+See the [Images Settings](https://our.umbraco.com/Documentation/Reference/Configuration/ImagingSettings/#browser-max-age) article for more information.
+
+```json  
+"Umbraco": {
+    "CMS": {
+      "Imaging": {
+        "Cache": {
+          "BrowserMaxAge": "365.00:00:00",
+          "CacheMaxAge": "365.00:00:00",
+          "CachedNameLength": 8,
+          "CacheFolder": "~/umbraco/Data/TEMP/MediaCache"
+        },
+        "Resize": {
+          "MaxWidth": 5000,
+          "MaxHeight": 5000
+        }
+      }
+    }
+  }
+```
```

Comment:
This is duplicated from the [linked article](https://our.umbraco.com/Documentation/Reference/Configuration/ImagingSettings/) - consider removing it to avoid stale content.

```suggestion
```